tests/queries/0_stateless/02944_dynamically_change_filesystem_cache_size.reference

```diff
@@ -1,20 +1,40 @@
-100	10	10	1	10	0	0	0	0	/var/lib/clickhouse/filesystem_caches/s3_cache_02944/	5	5000	0	16
+100	10	10	1	10	0	0	0	0	/var/lib/clickhouse/filesystem_caches/s3_cache_02944_lru/	5	5000	0	16
```

Comment:
Maybe include the policy in the description?

Reply:
I need to rewrite `FileCacheSettings` to `BaseSettings`, then all new settings would automatically end up in `system.filesystem_cache_settings`, I will do that in a separate PR.

Reply:
https://github.com/ClickHouse/ClickHouse/pull/75452